plugin-locationcomponent/src/test/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImplTest.kt

```diff
@@ -202,22 +202,6 @@ class LocationComponentPluginImplTest {
     assertEquals(mockLocationProvider, locationComponentPlugin.getLocationProvider())
   }
 
-  @Test
-  fun testOnStyleChanged() {
-    every { locationPuckManager.isLayerInitialised() } returns false
-    preparePluginInitialisationWithEnabled()
-    verify(exactly = 2) { locationPuckManager.isLayerInitialised() }
-    verify(exactly = 1) { locationPuckManager.initialize(style) }
-  }
-
-  @Test
-  fun testOnStyleChangedWhilePuckManagerInitialised() {
-    every { locationPuckManager.isLayerInitialised() } returns true
-    preparePluginInitialisationWithEnabled()
-    verify(exactly = 2) { locationPuckManager.isLayerInitialised() }
-    verify(exactly = 0) { locationPuckManager.initialize(style) }
-  }
-
```

Comment:
Just curious why those tests were fully removed but not have expected values adjusted. I tend to believe those tests did check some concrete scenario that could still be valid or?..

Reply:
Both tests checked that condition `isLayerInitialised` is checked, then puck manager initialized accordingly. 
Since this condition is now checked inside puck manager itself - test is not needed anymore.